doc/news/changes/incompatibilities/20170514DanielArndt-1

```diff
@@ -0,0 +1,6 @@
+Changed: The deprecated functions BlockSparseMatrixEZ<Number>::n_rows()
+and BlockSparseMatrixEZ<Number>::n_cols() have been removed.
+Use BlockSparseMatrixEZ<Number>::m() and
+BlockSparseMatrixEZ<Number>::n() instead.
```

Comment:
doxygen at least in the past wasn't able to correctly reference functions if you specify `<Number>` template arguments. So we just reference members without the template argument.